### ISSUE TITLE
Keep aspect ratio when resizing to specific dimension.

### ISSUE THREAD

**Issue by keyur1610** [Lymphatus/caesium-image-compressor#164]
When resizing images to a specific dimension, user should be able to set either height or width to 'Auto'.

For example, if I set the width to 1200px then I should have the option to set the height to 'Auto' so that resizing does not change the aspect ratio. 

To be clear, using 'long edge' or 'short edge' does not achieve this same behaviour in bulk resizing.

I was able to do what I described in the past version, a couple of years ago.

**Comment by flansch** [comment]
I sometimes have the requirement to bring a batch of pictures all to the same width (i.e. columnn width on a website), regardless of their orientation. So I also would be glad to see this implemented.

**Comment by vitalii-onopriienko** [comment]
Without this feature, this application has become useless for me. Such a shame. The best option for image bulk optimization.

**Comment by Lymphatus** [comment]
I think I'll start working on this with the upcoming (2.6.0) version.

**Comment by criquio** [comment]
I was using version 1.7.0 which has this feature. Today, I downloaded version 2.5.1 and I noticed it does not have it. I got a bit upset because it's one of the most important feature in this kind of software. The compress power in newer version, in other hand, is much more better. I compressed a png file in both versions and it got 5 times smaller in the newer one. The original size was 166 kb. The older version compressed it to 51.88 kb while the newer one compressed it to 10.4 kb. I also reduced it's dimensions to half the original one.
